digestparser/medium_post.py

```diff
@@ -124,11 +124,12 @@ def digest_medium_content_format(digest_config):
     return content_format
 
 
-def build_medium_content(file_name, digest_config=None, jats_file_name=None):
+def build_medium_content(file_name, temp_dir='tmp', digest_config=None, 
+                         jats_file_name=None, image_file_name=None):
     "build Medium content from a DOCX input file"
```

Comment:
What is `Medium content`? is this a format?

Reply:
nice website but I'm not sure I understand. Does this function create a formatted version of the docx contents that https://medium.com expects to consume somehow?

Reply:
Correct, the digest is formatted into HTML with some additional metadata, and that will be posted to the Medium API.